src/evo/simplifiedmns.cpp

```diff
@@ -233,6 +234,7 @@ void CSimplifiedMNListDiff::ToJson(UniValue& obj, bool extended) const
             obj.pushKV("merkleRootQuorums", cbTxPayload.merkleRootQuorums.ToString());
         }
     }
+    obj.pushKV("v19activationHeight", v19activationHeight);
```

Comment:
can we use a bit more meaningful name here that would provide some context maybe?... how about `basicBLSActivationHeight`?